articles/cognitive-services/Computer-vision/computer-vision-how-to-install-containers.md

```diff
@@ -66,13 +66,13 @@ There are several ways to authenticate with the private container registry for C
 
 Use the [docker login](https://docs.docker.com/engine/reference/commandline/login/) command, as shown in the following example, to log into `containerpreview.azurecr.io`, the private container registry for Cognitive Services Containers. Replace *\<username\>* with the user name and *\<password\>* with the password provided in the credentials you received from the Azure Cognitive Services team.
 
-```docker
+```DockerFile
 docker login containerpreview.azurecr.io -u <username> -p <password>
 ```
 
 If you have secured your credentials in a text file, you can concatenate the contents of that text file, using the `cat` command, to the `docker login` command as shown in the following example. Replace *\<passwordFile\>* with the path and name of the text file containing the password and *\<username\>* with the user name provided in your credentials.
 
-```docker
+```DockerFile
```

Comment:
note: this still probably isn't a correct language block because this is actually a block of commands